helpers/fees.ts

```diff
@@ -28,46 +18,37 @@ export const getFee = async (
   }
 
   if (type === "zevm") {
-    return fetchZEVMFees(network, provider, hre);
+    return fetchZEVMFees(network);
   } else if (type === "ccm") {
-    return fetchCCMFees(network, hre);
+    return fetchCCMFees(network);
   }
 };
 
 const formatTo18Decimals = (n: any) => parseFloat(formatEther(n)).toFixed(18);
 
-export const fetchZEVMFees = async (
-  network: string,
-  provider: any,
-  hre: HardhatRuntimeEnvironment
-) => {
+export const fetchZEVMFees = async (network: string) => {
+  const { url } = getHardhatConfigNetworks()["zeta_testnet"] as any;
+
+  const provider = new ethers.providers.JsonRpcProvider(url);
   const btcZRC20 = "0x65a45c57636f9BcCeD4fe193A602008578BcA90b"; // TODO: use getAddress("zrc20", "btc_testnet") when available
   const zrc20Address =
-    network === "btc_testnet" ? btcZRC20 : getAddress("zrc20", network);
+    network === "btc_testnet" ? btcZRC20 : getAddress("zrc20", network as any);
   if (!zrc20Address) return;
 
-  const contract = new hre.ethers.Contract(zrc20Address, ZRC20.abi, provider);
-
-  const gasFee = hre.ethers.BigNumber.from(
-    (await contract.withdrawGasFee())[1]
-  );
-  const protocolFee = hre.ethers.BigNumber.from(
-    await contract.PROTOCOL_FLAT_FEE()
-  );
+  const contract = new ethers.Contract(zrc20Address, ZRC20.abi, provider);
+  const gasFee = ethers.BigNumber.from((await contract.withdrawGasFee())[1]);
```

Comment:
can you split this in two lines with some aux variable? is confusing the await(await()[index])

Reply:
https://github.com/zeta-chain/toolkit/pull/30/commits/7509a24e1ca797f1d3445f619bdd16d2b098037a